lib/components/settings/widgets/settings_card.dart

```diff
@@ -36,7 +37,7 @@ class SettingsCard extends SettingsElementModel {
     this.onToggle,
     required this.value,
   }) : super(type: SettingsElementModelType.SWITCH, key: key) {
-    this.content = null;
+    content = null;
```

Comment:
continuing on previous comments, instead of initializing these inside constructor body assign them before calling super.
example:
}) : content = null, super(...);